### PR TITLE
[FUSEQE-9071] Dockerfile for UI tests

### PR DESCRIPTION
**Pull request by mkralik3**

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: docker/ui/settings.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <profiles>
+        <profile>
+            <id>syndesisRepo</id>
+            <repositories>
+                <repository>
+                    <id>fuse-all</id>
+                    <name>fuse-all</name>
+                    <url>http://nexus.fuse-qe.eng.rdu2.redhat.com/repository/fuse-all/</url>
```
Comment (avano) [inline docker/ui/settings.xml:11]: is it needed? it would be good not to leak the internal stuff

Comment (mkralik3) [reply docker/ui/settings.xml:11]: it is needed for apicurio UI steps dependency. However,
🤔 the whole commit is old (I don't use settings.xml anymore), probably I didn't push the actual local rebase (I will fix it tomorrow). Anyway, when you look to the backport ( https://github.com/syndesisio/syndesis-qe/pull/1566/commits/2a19c68bc45bfeda02e062a72bab1e6ab5809e80#diff-f1a049876079bf56678d719d5a76e41fR76 ), I use that repo only for the apicurio artifact so the other dependencies are downloaded from the public central. Is that way ok?

Comment (avano) [reply docker/ui/settings.xml]: you can build apicurio the same way you are building syndesis and copy it in the local repo

Comment (mkralik3) [reply docker/ui/settings.xml]: I didn't know that apicurio tests are in github, ok, I will try to use it.

**Comment by mkralik3** [comment]
@avano  Can you please look at it whether it needs some changes or can I merge the pr ?

**Pull request merged**